script/core/deployAllo.sh

```diff
@@ -0,0 +1,130 @@
+#!/bin/bash
+
+# Function to dynamically map chains to variables
+# Relevant issue that would deprecate this script if solved: https://github.com/foundry-rs/foundry/issues/7726
+deploy() {
+  chain=$1
+  case "$chain" in
```

Comment:
need to add `local` here

Reply:
@ilpepepig You can copy what I did: https://github.com/allo-protocol/allo-v2.1/pull/2/commits/384db66003eb4b83ae7dbebb2a7d04ed64843e73